master/buildbot/scripts/devproxy.py

```diff
@@ -148,15 +148,24 @@ def connection_error(self, error):
 
     async def fetch_config_from_upstream(self):
         async with self.session.get(self.next_url) as request:
-            index = await request.content.read()
+            html = await request.content.read()
             if request.status != 200:
-                raise RuntimeError("Unable to fetch buildbot config: " + index.decode())
+                raise RuntimeError("Unable to fetch buildbot config: " + html.decode())
         # hack to parse the configjson from upstream buildbot config
         start_delimiter = b'angular.module("buildbot_config", []).constant("config", '
```

Comment:
Could the json code be extracted into a separate function so that we can add a unit test for it? Unit tests would live in master/buildbot/test/unit/test_scripts_devproxy.py file